google/cloud/spanner/internal/transaction_impl_test.cc

```diff
@@ -92,11 +92,12 @@ class Client {
 
  private:
   ResultSet Read(SessionHolder& session,
-                 absl::optional<TransactionSelector>& selector,
+                 StatusOr<TransactionSelector>& selector,
                  std::int64_t seqno, std::string const& table,
                  KeySet const& keys, std::vector<std::string> const& columns);
 
   Mode mode_;
+  const Status failed_txn_status_{StatusCode::kInternal, "Bad transaction"};
```

Comment:
If we're going to add a `const` member, we might as well constify `mode_` too.  Otherwise, drop `const`.

Alternatively, this status could be a local constant in `Read()`.